src/sentry/api/serializers/models/project.py

```diff
@@ -352,6 +352,7 @@ def serialize(self, obj, attrs, user):
             "dateCreated": obj.date_added,
             "firstEvent": obj.first_event,
             "firstTransactionEvent": True if obj.flags.has_transactions else False,
+            "hasSessions": True if obj.flags.has_sessions else False,
```

Comment:
`bool(obj.flags.has_sessions)` (or maybe can just use it directly?)

Reply:
Should I change the line above too 👀

Reply:
(bool was required, otherwise it gives me the bit object. Could probably use `.has_sessions.is_set` though)